docs/nss.md

```diff
@@ -0,0 +1,21 @@
+# NSS
+
+## Incompatible NSS/JSS Pairings
+
+The minimum NSS version required by JSS is v3.30.
+
+### JSS v4.6.3
+
+JSS v4.6.3 doesn't support NSS versions between v3.47 and v3.50, inclusive.
```

Comment:
Should we specify the `Conflicts` with those NSS versions in `jss.spec`?

Reply:
I guess we could've for that version. :-) But for this version, we've fixed the incompatibilities. And it only was incompatible if you tried building (tests would've built) or if you tried using CMAC or KBKDF. They're not popular -- we're the only ones that use them (and these versions of NSS are really new) -- so "meh"? :-) I could push an update to Fedora if you'd like though.

Reply:
If it's only a build issue it's probably not necessary since we control Fedora builds (unless someone accidentally builds with the wrong NSS version). If it's a potential runtime issue (i.e. user installs the wrong JSS/NSS combinantion) we might want to add the `Conflicts`.